### PR TITLE
Rework structure of project

### PR DESCRIPTION
**Pull request by marsofandrew**
now module only consists client modules

### CODE REVIEW

File: playbooks/examples/fileInterface.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  tasks:
+    - name: nfsServer creation and deletion
+      fileInterface:
+        login:
+          #host: "{{ host }}"
+          #username: "{{ username }}"
+          #password: "{{ password }}"
+          host: 192.168.70.217
```
[the remark this answers is not in the document]
Comment (marsofandrew) [reply playbooks/examples/fileInterface.yml]: Yes we can

File: library/fileInterface.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+from dellemc_unity_sdk import validator
+from dellemc_unity_sdk.unity import Unity
+from dellemc_unity_sdk import runner
+
+optional_list = {
+    'netmask','v6PrefixLength','gateway',
+    'vlanId','isPreferred','role'}
+
+def _exception_about_parameters(supported_parameters):
+    return False, 'You did not input required parameters or inputted unsupported parameter, ' \
```
[the remark this answers is not in the document]
Comment (marsofandrew) [reply library/fileInterface.py]: We remake it in version 0.4, now we are testing it

File: playbooks/examples/nfsServer_create_and_delete.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - name: nfsServer creation and deletion
+      nfsServer:
+        login:
+          #host: "{{ host }}"
+          #username: "{{ username }}"
+          #password: "{{ password }}"
+          host: 192.168.70.217
+          username: admin
+          password: Password123!
+        #create:
```
[the remark this answers is not in the document]
Comment (marsofandrew) [reply playbooks/examples/nfsServer_create_and_delete.yml]: So, today we have got the same decision. It's better to have a plabook for each action

**Review by Mityushin — APPROVED** [review]
Need remove unused imports

**Review by vksychev — APPROVED** [review]

**Pull request merged**